wp-content/mu-plugins/pantheon/pantheon-try-gutenberg-mods.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+  * Should we proceed with adding the
+  * return to Pantheon button?
+  *
+  * Only if:
+  * We are on a Pantheon subdomain and
+  * "RETURN_TO_PANTHEON_BUTTON" is not false
+  */
+$mod_try_gutenberg = apply_filters( 'show_pantheon_try_gutenberg_mods', ( ! defined('DISABLE_PANTHEON_TRY_GUTENBERG_MODS') || DISABLE_PANTHEON_TRY_GUTENBERG_MODS ) );
```

Comment:
@ari-gold I updated it so you can use 

```php
if ( ! defined( 'DISABLE_PANTHEON_TRY_GUTENBERG_MODS' ) ) {
	define( 'DISABLE_PANTHEON_TRY_GUTENBERG_MODS', false );
}
```
in `wp-config.php` to disable, similarly to the login form return to Pantheon button.